plugin-scalebar/src/main/java/com/mapbox/pluginscalebar/ScaleBarWidget.java

```diff
@@ -0,0 +1,161 @@
+package com.mapbox.pluginscalebar;
+
+import android.content.Context;
+import android.graphics.Canvas;
+import android.graphics.Paint;
+import android.support.annotation.ColorInt;
+import android.util.Pair;
+import android.view.View;
+
+import java.util.ArrayList;
+import java.util.Locale;
+
+/**
+ * The scale widget is a visually representation of the scale plugin.
+ */
+class ScaleBarWidget extends View {
+  private int textColor;
+  private int primaryColor;
+  private int secondaryColor;
+  private int width;
```

Comment:
the width field is actually the mapView width as seen in https://github.com/mapbox/mapbox-plugins-android/pull/955/files#diff-b120ab4a1074a16f4fbb0d58dd43150dR106. Renaming it to `mapViewWidth` (and the setter) makes that instantly clear to the reader of the code.